src/features/content-explorer/content-explorer/__tests__/ContentExplorer.test.js

```diff
@@ -739,6 +749,13 @@ describe('features/content-explorer/content-explorer/ContentExplorer', () => {
             expect(result).toStrictEqual({});
         });
 
+        test('should remove items from autoSelectedItems when unselectAll is called', () => {
+            const wrapper = renderComponent({ items });
+            wrapper.setState({ selectedItems });
+            const result = wrapper.instance().unselectAll();
+            expect(result).toStrictEqual({});
+        });
```

Comment:
are we okay to remove this test? it looks the same as the one above

Reply:
Ah yes, missed removing this one while removing the additional prop.